tests/testthat/test-Design-methods.R

```diff
@@ -92,3 +92,99 @@ test_that("NextBestInfTheory produces consistent results with a dataset", {
   expect_equal(result@toxAtDosesSelected, rep(1L, 5))
   expect_snapshot(result@meanFit)
 })
+
+test_that("reporting labels and logicals are correctly returned using the recursive function in simulate", {
+  my_data <- h_get_data(placebo = FALSE)
+  model <- LogisticLogNormal(
+    mean = c(-0.85, 1),
+    cov =
+      matrix(c(1, -0.5, -0.5, 1),
+        nrow = 2
+      ),
+    ref_dose = 56
+  )
```

Comment:
```suggestion
  model <- h_get_logistic_normal()
```
and also the same comment in other places.